### PR TITLE
Update Application.xml

### PR DESCRIPTION
**Pull request by RussKie**
## Summary

Fix incorrect rendering of the remarks
image

### CODE REVIEW

File: xml/System.Windows.Forms/Application.xml [hunk 1]
```diff
@@ -1585,89 +1585,83 @@ static class Program
...
              The system text scale factor will be applied to the font, i.e. if the default font is set to "Calibri, 11f"
              and the text scale factor is set to 150% the resulting default font will be set to "Calibri, 16.5f".
-            
-
-
-             Users can adjust text scale with the Make text bigger slider on the Settings -&gt; Ease of Access -&gt; Vision/Display screen.
-            
-
 
+             Users can adjust text scale with the Make text bigger slider on the Settings -> Ease of Access -> Vision/Display screen.
```
Comment (gewarren) [inline xml/System.Windows.Forms/Application.xml:1630]:
```suggestion
You can adjust text scale with the **Make text bigger** slider on the **Settings** > **Ease of Access** > **Vision/Display** screen.
```

File: xml/System.Windows.Forms/Application.xml [hunk 2]
```diff
@@ -1585,89 +1585,83 @@ static class Program
...
+      <Docs>
+        <param name="font">The font to be used as a default across the application.</param>
+        <summary>Sets the default <see cref="T:System.Drawing.Font" /> for the process.</summary>
+        <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
-
              The system text scale factor will be applied to the font, i.e. if the default font is set to "Calibri, 11f"
```
Comment (gewarren) [inline xml/System.Windows.Forms/Application.xml:1627]:
```suggestion
The system text scale factor is applied to the font. For example, if the default font is set to "Calibri, 11f"
```

File: xml/System.Windows.Forms/Application.xml [hunk 3]
```diff
@@ -1585,89 +1585,83 @@ static class Program
...
              and the text scale factor is set to 150% the resulting default font will be set to "Calibri, 16.5f".
```
Comment (gewarren) [inline xml/System.Windows.Forms/Application.xml:1628]:
```suggestion
and the text scale factor is set to 150%, the resulting default font is set to "Calibri, 16.5f".
```

**Review by gewarren — APPROVED** [review]
The problem is the indentation.

**Comment by RussKie** [comment]
> The problem is the indentation.

This is [our xmldocs](https://github.com/dotnet/winforms/blob/08b29bcd36af441b653b89a32a9a4ce285a35ae9/src/System.Windows.Forms/src/System/Windows/Forms/Application.cs#L1245-L1263):
image

Feels like we have some kind of bug in the transformation pipeline which creates such wild indentations. Would you know where to raise this?

**Pull request merged**